crates/jstzd/src/task/octez_node.rs

```diff
@@ -0,0 +1,264 @@
+use super::Task;
+use anyhow::Result;
+use async_dropper::AsyncDrop;
+use async_trait::async_trait;
+use std::{fs::File, path::PathBuf, sync::Arc};
+use tokio::{runtime::Handle, sync::RwLock};
+
+#[cfg(test)]
+use tests::{Child, InnerOctezNode};
+
+#[cfg(not(test))]
+use octez::OctezNode as InnerOctezNode;
+#[cfg(not(test))]
+use std::process::Child;
+
+pub struct OctezNodeConfig {
+    binary_path: PathBuf,
+    data_dir: PathBuf,
+    network: String,
+    http_endpoint: String,
+    rpc_endpoint: String,
+    num_connections: u32,
+    log_file: PathBuf,
+    options: Vec<String>,
```

Comment:
I'll do this after #565 is merged so that I don't need to convert options to strings so that they can be passed into the current octez node implementation.